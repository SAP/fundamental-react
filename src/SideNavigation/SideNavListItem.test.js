import { mount } from 'enzyme';
import React from 'react';
import { SideNav } from '../';

describe('<SideNavListItem />', () => {

    describe('Prop spreading', () => {
        test('should allow props to be spread to the SideNav.ListItem component', () => {
            const element = mount(<SideNav.ListItem data-sample='Sample' />);

            expect(
                element.getDOMNode().attributes['data-sample'].value
            ).toBe('Sample');
        });
    });
});
