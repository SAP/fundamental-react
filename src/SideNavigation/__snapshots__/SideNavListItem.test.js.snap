// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SideNavListItem /> create side navigation 1`] = `
<li
  className="fd-nested-list__item"
>
  <a
    className="fd-nested-list__link"
    href="#"
    onClick={[Function]}
  >
    <span
      className="fd-nested-list__title"
    >
      Item 1
    </span>
  </a>
</li>
`;
